src/styles/blocks/header.scss

```diff
@@ -0,0 +1,83 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 0 50px;
+
+  background: #fff;
+  box-shadow: 0 2px 4px 0 #0000000d;
+
+  &__link {
+    width: 40px;
+    height: 40px;
+    display: block;
+
+    img {
+      width: 100%;
+      height: 100%;
+    }
+  }
+
+  &__nav {
+    display: block;
+    align-items: center;
+
+    &-list {
+      display: flex;
```

Comment:
So if you use separate block for nav, there the should be separated file for navigation